### PR TITLE
Clarify steps for adding and verifying domains in SSO docs

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

This PR includes clearer steps for adding a domain, including best practice for setting up the TXT record. This is in response to several support tickets where users encountered an issue with setting up their TXT record properly in their DNS settings. The modified file is a template that is used in Docker Admin and Docker Hub administration docs.

Pages using the template:
- [Admin - Company management - Domain management](https://deploy-preview-17803--docsdocker.netlify.app/admin/company/settings/domains/)
- [Admin - Company management - Configure SSO](https://deploy-preview-17803--docsdocker.netlify.app/admin/company/settings/sso-configuration/#step-one-add-and-verify-your-domain)
- [Admin - Org management - Configure SSO](https://deploy-preview-17803--docsdocker.netlify.app/admin/organization/security-settings/sso-configuration/)
- [Administration and security - Configure SSO](https://deploy-preview-17803--docsdocker.netlify.app/admin/organization/security-settings/sso-configuration/)

### Related issues (optional)

JIRA: ENGDOCS-1497

### CODE REVIEW

**Review by craig-osterhout — APPROVED** [review]
LGTM. Marked a few preexisting issues that are non blockers.

File: _includes/admin-domains.md [hunk 1]
```diff
@@ -16,10 +16,18 @@
...
-4. Once you have waited 72 hours for the TXT Record verification, you can then select **Verify** next to the domain you've added, and follow the on-screen instructions.
\ No newline at end of file
+5. Add a TXT record to your DNS settings. The steps to do so may vary depending on your DNS provider.
+
+   >**Note**
+   >
+   > When you add the TXT record, ensure that you create the record on the root of the zone. This is typically denoted with the `@` name for the record.
+
+6. Use the TXT Record Value that you copied in the prior step and add this into the corresponding field of the record you create.
+7. Once you have waited 72 hours for the TXT Record verification, you can then select **Verify** next to the domain you've added, and follow the on-screen instructions.
```
[the remark this answers is not in the document]
Comment (stephaurelio) [reply _includes/admin-domains.md:33]: Not a UI text so will be updating it with that suggestion. Thanks!

**Review by craig-osterhout — APPROVED** [review]

File: _includes/admin-domains.md [hunk 2]
```diff
@@ -16,10 +16,18 @@
...
     >
     > Format your domains without protocol or www information, for example, `yourcompany.example`. This should include all email domains and subdomains users will use to access Docker, for example `yourcompany.example` and `us.yourcompany.example`. Public domains such as `gmail.com`, `outlook.com`, etc. aren’t permitted. Also, the email domain should be set as the primary email.
 
-4. Once you have waited 72 hours for the TXT Record verification, you can then select **Verify** next to the domain you've added, and follow the on-screen instructions.
\ No newline at end of file
+5. Add a TXT record to your DNS settings. The steps to do so may vary depending on your DNS provider.
+
+   >**Note**
+   >
+   > When you add the TXT record, ensure that you create the record on the root of the zone. This is typically denoted with the `@` name for the record.
```
Comment (stephaurelio) [inline _includes/admin-domains.md:30]: @ivan-californias Just wanted to verify with you that this wording looks good since this seemed to be the area some users were struggling with based on some Slack threads where you found the record wasn't created on the root.

Comment (ivan-californias) [reply _includes/admin-domains.md:30]: **suggestion:** There are some cases where the customer might not want to use the root of their domain, ie `us.yourcompany.com`. I'd rephrase this block to something similar to:

> Make sure that TXT record name that you create on your DNS matches with the domain you registered on Docker (Step4). For example, if you registered the subdomain `us.yourcompany.com` you need to create a TXT record within the same name/zone `us`. If you registered a root domain `yourcompany.com` you need to create a TXT record on the root zone, which is typically denoted with the `@` as name for the record.

File: _includes/admin-domains.md [hunk 3]
```diff
@@ -16,10 +16,18 @@
...
+
+6. Use the **TXT Record Value** that you copied in the prior step and add this into the corresponding field of the record you create.
```
Comment (ivan-californias) [inline _includes/admin-domains.md:32]: **suggestion:** Remove step 6 as seems repetitive of step 5

File: _includes/admin-domains.md (hunk 2 above)
Comment (stephaurelio) [reply _includes/admin-domains.md:30]: Sounds good! Will update.

File: _includes/admin-domains.md [hunk 4]
```diff
@@ -16,10 +16,18 @@
 
 1. Sign in to {{ product_link }}{: target="_blank" rel="noopener" class="_"}.
 2. {{ domain_navigation }}
-3. Select **Add a domain** and continue with the on-screen instructions to add the TXT Record Value to your domain name system (DNS).
+3. Select **Add a domain**.
+4. Continue with the on-screen instructions to get the **TXT Record Value**. Copy this value to paste into a TXT record that you create in your Domain Name System (DNS) settings.
 
     >**Note**
     >
     > Format your domains without protocol or www information, for example, `yourcompany.example`. This should include all email domains and subdomains users will use to access Docker, for example `yourcompany.example` and `us.yourcompany.example`. Public domains such as `gmail.com`, `outlook.com`, etc. aren’t permitted. Also, the email domain should be set as the primary email.
```
[the remark this answers is not in the document]
Comment (stephaurelio) [reply _includes/admin-domains.md:24]: There was feedback on a previous PR to change these to `yourcompany.example` but I get what you mean! Apparently it is some best practice that we haven't listed in the style guide. Not sure about the primary email portion as that was part of the original note so I'm not sure either what prompted the inclusion here, so if it isn't helpful I can remove it.

**Review by craig-osterhout — APPROVED** [review]
